vortex-io/src/dispatcher/wasm.rs

```diff
@@ -3,11 +3,10 @@
 use std::future::Future;
 
 use futures::channel::oneshot;
-use futures::channel::oneshot::Receiver;
 use vortex_error::{vortex_panic, VortexResult};
 use wasm_bindgen_futures::wasm_bindgen::__rt::Start;
 
-use crate::{Dispatch, JoinHandle as VortexJoinHandle};
+use super::{Dispatch, JoinHandle as VortexJoinHandle};
```

Comment:
if you use `crate` then you only get the pub re-export of `JoinHandle` whose constructor is not pub